configure.bat

```diff
@@ -28,19 +28,11 @@ exit /B %errorlevel%
     )
     set envfile=
 
-    set all_set=1
-    if not defined OPENSSL_DIR set all_set=0
-
-    if %all_set%==1 (
-        @echo - Using OpenSSL: %OPENSSL_DIR%
-    ) else (
-        @echo At least one of the following variables were not set:
-        @echo     - OPENSSL_DIR: %OPENSSL_DIR%
-        @echo Please define them using by creating a "env.bat" file containing:
-        @echo     @set OPENSSL_DIR=^<your OpenSSL directory^>
-        exit /B 1
+    if not defined openssl_dir (
+        echo [INFO] "OPENSSL_DIR" environment variable is not defined. If meson
+        echo        does not find OpenSSL, consider setting it to your current
+        echo        OPENSSL library directory.
```

Comment:
Why not maintain the original message and break? Actually, maybe changing it for something to make it clear that is only about OpenSSL like: 

```suggestion
        @echo OPENSSL_DIR variable was not set:
        @echo     - OPENSSL_DIR: %OPENSSL_DIR%
        @echo Please define it by creating a "env.bat" file containing:
        @echo     @set OPENSSL_DIR=^<your OpenSSL directory^>
        exit /B 1
```

Reply:
Because this makes it looks like it is obligatory. OpenSSL_DIR is not meant to be the default, it is optional.

Reply:
I mean, it is not even "looks like". It is literally mandating the user to define it.